flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/queryConfig.scala

```diff
@@ -37,6 +37,13 @@ class BatchQueryConfig private[table] extends QueryConfig
 class StreamQueryConfig private[table] extends QueryConfig {
 
   /**
+    * The non-windowed groupby aggregate update the calculation result according a configuration of
+    * time interval. By default non-windowed groupby aggregate will update calculation result each
+    * row.
+    */
+  private var unboundedAggregateUpdateInterval: Time = Time.milliseconds(0)
```

Comment:
I think we need to achieve separately, i.e., proc-time non-windowed aggregations using `process-time timer.` and event-time non-windowed aggregations using event-time timer. `updateInterval` also make sense, because `updateInterval` is relative to the last emit time. i.e., emitTime = current(Proc/Row)Time+updateInterval. (Calculate emitTime when the data arrives).